eland/ml/transformers/sklearn.py

```diff
@@ -86,8 +86,12 @@ def build_tree_node(
             ):  # classification requires more than one value, so assume regression
                 leaf_value = [float(value[0][0])]
             else:
-                # the classification value, which is the index of the largest value
-                leaf_value = [float(np.argmax(value))]
+                # the classification value
+                # DecisionTreeClassifiers simply use normalize (dividing predicted values by sum)
+                # We use softMax, to get our probabilities as close as possible, store log value here
+                leaf_value = [
+                    -10000000 if n == 0 else math.log(float(n)) for n in value[0]
```

Comment:
Feels like we should be careful about negative values. Shouldn't crop up but we can be defensive. Either error out or treat as zero, e.g.
```suggestion
                    -10000000 if n <= 0 else math.log(float(n)) for n in value[0]
```